app/lib/actions/candlepin/owner/import_products.rb

```diff
@@ -8,10 +8,12 @@ class ImportProducts < Candlepin::Abstract
 
         def run
           organization = ::Organization.find(input[:organization_id])
+          updated_content = []
```

Comment:
I think this assignment is not necessary

Reply:
I think it is. Are you sure ?

Reply:
you are checking `if updated_content.present?` down below and that returns false if the object is nil so it's safe to do